src/test/java/edu/cornell/kfs/paymentworks/util/PaymentWorksUtilTest.java

```diff
@@ -0,0 +1,112 @@
+			.append("\",\"to_value\":\"").append(PAYMENT_WORKS_FIELD_TO_VALUE).append("\"}]}");
+		assertEquals("Conversion wasn't what was expected", builder.toString(), paymentWorksUtil.pojoToJsonString(buildPaymentWorksFieldChangesDTO()));
+	}
+
+	@Test
+	public void testConvertFieldArrayToMapPaymentWorksCustomFieldsDTO() {
+		PaymentWorksCustomFieldsDTO customFields = new PaymentWorksCustomFieldsDTO();
+		List<PaymentWorksCustomFieldDTO> custom_fields = new ArrayList<PaymentWorksCustomFieldDTO>();
+		PaymentWorksCustomFieldDTO dto = new PaymentWorksCustomFieldDTO();
+		String label = "the label";
+		String value = "super cool value";
+		dto.setField_id("12344");
+		dto.setField_label(label);
+		dto.setField_value(value);
+		custom_fields.add(dto);
+		customFields.setCustom_fields(custom_fields);
+		Map<String, String> results = paymentWorksUtil.convertFieldArrayToMap(customFields);
+		assertEquals("the value was not what we expected", value, results.get(label));
+	}
+
+	@Test
+	public void testConvertFieldArrayToMapPaymentWorksFieldChangesDTO() {
+		Map<String, String> results = paymentWorksUtil.convertFieldArrayToMap(buildPaymentWorksFieldChangesDTO());
+		assertEquals("the value was not what we expected", PAYMENT_WORKS_FIELD_TO_VALUE, results.get(PAYMENT_WORKS_FIELD_NAME));
+	}
+
+	@Test
+	public void testConvertFieldArrayToMapFromValues() {
+		Map<String, String> results = paymentWorksUtil.convertFieldArrayToMapFromValues(buildPaymentWorksFieldChangesDTO());
+		assertEquals("the value was not what we expected", PAYMENT_WORKS_FIELD_FROM_VALUE, results.get(PAYMENT_WORKS_FIELD_NAME));
+	}
+	
+	private PaymentWorksFieldChangesDTO buildPaymentWorksFieldChangesDTO() {
+		PaymentWorksFieldChangesDTO fieldChanges = new PaymentWorksFieldChangesDTO();
+		List<PaymentWorksFieldChangeDTO> field_changes = new ArrayList<PaymentWorksFieldChangeDTO>();
+		PaymentWorksFieldChangeDTO dto = new PaymentWorksFieldChangeDTO();
+		dto.setField_name(PAYMENT_WORKS_FIELD_NAME);
+		dto.setFrom_value(PAYMENT_WORKS_FIELD_FROM_VALUE);
+		dto.setTo_value(PAYMENT_WORKS_FIELD_TO_VALUE);
+		field_changes.add(dto);
+		fieldChanges.setField_changes(field_changes);
+		return fieldChanges;
+	}
+
+	@Test
+	public void testConvertPhoneNumber_withAreaAndExtension() {
+		String expected="234-567-9012-4567";
+        assertEquals("Conversion did not happen as expected", expected, paymentWorksUtil.convertPhoneNumber("2345679012;ext4567"));
+        assertEquals("Conversion did not happen as expected", expected, paymentWorksUtil.convertPhoneNumber("234-567-9012-4567"));
+        assertEquals("Conversion did not happen as expected", expected, paymentWorksUtil.convertPhoneNumber("1-234-567-9012;ext4567"));
+        assertEquals("Conversion did not happen as expected", expected, paymentWorksUtil.convertPhoneNumber("12345679012--4567"));
+	}
+	
+	@Test
+	public void testConvertPhoneNumber_wtihAreaCode() {
```

Comment:
misspelling in method name (and I would remove _ from these method names and just go with camel case